frontend/src/pages/GuestAccessPage.tsx

```diff
@@ -47,10 +47,10 @@ const GuestAccessPage = () => {
         <Col>
           <Formik
             initialValues={{
-              agency: null,
-              role: null,
+              agency: undefined,
```

Comment:
in Yup undefined better represents initial state.

Reply:
I believe my tests check for null state but it's a quick fix - just in case you end up running them. Will change them on my end for next PR!